### ISSUE TITLE
[2.9 ACRA] crash releasing audio recorder

### ISSUE THREAD

**Issue by mikehardy** [ankidroid/Anki-Android#5521]
This one should be easy

https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/report-details/22f163c7-52c4-44a6-8de6-723cb52ecd32

```
java.lang.NullPointerException: Attempt to invoke virtual method 'void com.ichi2.anki.multimediacard.AudioView.notifyReleaseRecorder()' on a null object reference
at com.ichi2.anki.multimediacard.fields.BasicAudioRecordingFieldController.onFocusLost(BasicAudioRecordingFieldController.java:1)
at com.ichi2.anki.multimediacard.activity.MultimediaEditFieldActivity.onOptionsItemSelected(MultimediaEditFieldActivity.java:8)
at android.app.Activity.onMenuItemSelected(Activity.java:3644)
```

**Comment by david-allison-1** [comment]
I'm looking at handling this in the design for #5407

Cause: `IFieldController` is set to a member before being fully initialised.

https://github.com/ankidroid/Anki-Android/blob/de5b06c0b148c604f9fc3c00277554ee0e2ea2e7/AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.java#L115

https://github.com/ankidroid/Anki-Android/blob/de5b06c0b148c604f9fc3c00277554ee0e2ea2e7/AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.java#L137-L140

**Comment by david-allison-1** [comment]
Fixed by #5856/#5841, mFieldController can no longer be in an invalid state.
